tests/ui_tests/app/page/common/base_page.py

```diff
@@ -74,11 +74,6 @@ def __init__(
         default_loc_timeout: int = 15,
         **kwargs,
     ):
-        if any(str.isdigit(char) for char in path_template):
```

Comment:
why it's removed? false-positives?

Reply:
because of pages:
/job/1/1 - last index is for stdout
/job/1/2 - last index is for stderr